### PR TITLE
Add medasdigital

### PR DESCRIPTION
**Pull request by oxygene76**

### CODE REVIEW

File: medasdigital/chain.json [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+  "status": "live",
+  "network_type": "mainnet",
+  "website": "https://medas-digital.io/",
+  "pretty_name": "Medas Digital Network",
+  "chain_id": "medasdigital-1",
+  "bech32_prefix": "medas",
+  "daemon_name": "medasdigitald",
+  "node_home": "$HOME/.medasdigital",
+  "genesis": {
+    "genesis_url": "https://github.com/oxygene76/medasdigital/tree/main/networks/mainnet/genesis.json"
```
[the remark this answers is not in the document]
Comment (oxygene76) [reply medasdigital/chain.json]: Moved

File: medasdigital/assetlist.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+        "name": "Medas Digital",
+        "display": "medas",
+        "symbol": "MEDAS",
+        "logo_URIs": {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/medasdigital/images/medas.png",
+          "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/medasdigital/images/medas.svg"
+        },
+	"keywords": ["medas"]
+      }
+  }
```
Comment (JeremyParish69) [inline medasdigital/assetlist.json:29]: oh, didn't close the 'assets' array here

File: medasdigital/assetlist.json [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+            "aliases": []
+          }
+        ],
+        "base": "umedas",
+        "name": "Medas Digital",
+        "display": "medas",
+        "symbol": "MEDAS",
+        "logo_URIs": {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/medasdigital/images/medas.png",
+          "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/medasdigital/images/medas.svg"
```
Comment (JeremyParish69) [inline medasdigital/assetlist.json:25]: this svg image appears blank for me. Could there be an issue with this svg?

Comment (oxygene76) [reply medasdigital/assetlist.json]: Good Morning Jeremy, thank you for your assistance. I exchanged the svg.

Comment (JeremyParish69) [reply medasdigital/assetlist.json]: Good day, thanks for looking into it. But I'm Sorry the image does not match the png. the Png correctly has transparency, while the svg has grey where is should be transparent. Could you please correct?

Comment (oxygene76) [reply medasdigital/assetlist.json]: Hi, recreated SVG. In github it now looks good

**Pull request merged**